modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,116 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies implement an additional level of granularity with SLA tiering. Policies control access based on the following access structuring:
+
+* Resource-level policies
+Impose a limit on all requests that match the resource to which it is applied. All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+* SLA-based policies
+Impose a limit specified by the level of access granted to the requesting application. The Rate Limiting policy is SLA-based.
+
+When the API Gateway is configured to work as a Mule Cluster, the policy can be configured to work in a distributed manner.
+  
+  [PA: This seems out of place and does not make sense.]
```

Comment:
Yes, I don't think it goes here. We should just limit this to resource level policy that is the level of granularity that we add on top of API level (that is the default)

Reply:
Removed the sentence! Thanks!